docs/installation.md

```diff
@@ -0,0 +1,28 @@
+When running the local dev environment for the first time, a few extra commands need to be run.
+
+In addition you need to create a .env file in backend/ (copy backend/.env.example and update necessary values)
+
+Run `npm ci` in repo root.
+
+for frontend:
+
+1. `cd frontend`
+2. `npm ci`
+3. `npm run dev`
+
+for backend:
+
+1. `cd backend`
+2. `docker-compose up`
+
+then in another terminal:
+
+1. `cd backend`
+2. `npm ci`
+3. `sudo -u postgres psql` to open psql terminal
```

Comment:
Does this connect to the right PSQL instance?

Reply:
Yes